app/libs/twitter_processor.rb

```diff
@@ -0,0 +1,43 @@
+require "twitter"
+
+class TwitterProcessor
+  def initialize(company:, number_of_tweets:)
+    @twitter_link = company.twitter
+    @count = number_of_tweets
+    @client = initialize_twitter_client
+  end
+
+  def screen_name
+    @twitter_link.sub('https://twitter.com/', '') unless @twitter_link.blank?
+  end
+
+  def tweets
+    screen_name.blank? ? [] : scrape_tweets
+  rescue Twitter::Error
+    []
+  end
+
+  def followers
+    screen_name.blank? ? nil : scrape_followers
```

Comment:
no need to use ternary operator
```
def followers
  scrape_followers if screen_name.present? # returns nil if blank
end

def scrape_followers
  @client.user(screen_name).followers_count
rescue Twitter::Error # returns nil if error
end
```

Reply:
Rubocop retruns an offense this way, when there there is nothing set as handled return
`Do not suppress exceptions.`
  `rescue Twitter::Error`